### DISCUSSION TITLE
jinja2.exceptions.TemplateNotFound

### DISCUSSION THREAD

**Discussion by DevBeginner1** [pallets/flask#5297] [Q&A]
Hi.

I am discovering flask by following a tutorial on the web site : https://www.geeksforgeeks.org/flask-http-method/
I am trying to repeate the exercise but when I try to open the server after I have launched the Python script, it doesn't work. And I don't understand as I have created the "templates" directory, and put there the .html files.
Here are screenshots about my project and the issue.

Thanks for the help.

![page1](https://github.com/pallets/flask/assets/111196043/74566791-7ab6-4148-b49c-8b3fcebec02a)

![page2](https://github.com/pallets/flask/assets/111196043/e169f24e-75c3-44b3-b0e1-fe62f231ebe9)

![page3](https://github.com/pallets/flask/assets/111196043/ae217f00-9596-4f09-ac22-89647ab8fbc4)

![page4](https://github.com/pallets/flask/assets/111196043/b9004e6b-66f8-408c-b7a6-cae3036556f8)

![page5](https://github.com/pallets/flask/assets/111196043/4344fd61-84ae-4b89-8626-f2d134d222c3)

![page6](https://github.com/pallets/flask/assets/111196043/1669e1c2-4bd8-4119-999e-93531f1cbae5)

![page7](https://github.com/pallets/flask/assets/111196043/470c2fcb-1753-4079-b1fd-8ff8aee8a5a3)

**Comment by alexcybernetic** [comment]
try to move the files
.gitignore
app.py

out of the venv directory to the same directory level as the templates directory

Then adjust your run/execution command accordingly
E:\Python_Web_dev\blog2\venv\Scripts\python.exe E:\Python_Web_dev\blog2\app.py
